src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -542,10 +545,27 @@ private[cassandra] object CassandraJournal {
 }
 
 class FixedRetryPolicy(number: Int) extends RetryPolicy {
-  override def onUnavailable(statement: Statement, cl: ConsistencyLevel, requiredReplica: Int, aliveReplica: Int, nbRetry: Int): RetryDecision = retry(cl, nbRetry)
-  override def onWriteTimeout(statement: Statement, cl: ConsistencyLevel, writeType: WriteType, requiredAcks: Int, receivedAcks: Int, nbRetry: Int): RetryDecision = retry(cl, nbRetry)
-  override def onReadTimeout(statement: Statement, cl: ConsistencyLevel, requiredResponses: Int, receivedResponses: Int, dataRetrieved: Boolean, nbRetry: Int): RetryDecision = retry(cl, nbRetry)
-  override def onRequestError(statement: Statement, cl: ConsistencyLevel, cause: DriverException, nbRetry: Int): RetryDecision = tryNextHost(cl, nbRetry)
+  override def onUnavailable(statement: Statement, cl: ConsistencyLevel, requiredReplica: Int, aliveReplica: Int,
```

Comment:
Added, it's essentially explained in http://docs.datastax.com/en/developer/java-driver/3.1/manual/retries/